### DISCUSSION TITLE
My discord account was down.

### DISCUSSION THREAD

**Discussion by ghost** [1865-Mods/SMEE#16] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ghost** [comment]
@quick007, Tell the others that my discord account was locked out. idk what to do at this point in time, at least my mod is still hanging up to the stages.

**Reply by ghost** [reply]
and also the server will get deleted soon, so idk what to do right now.

**Reply by quick007** [reply]
alright, will do. thanks for telling me. Here an invite to the server if you create another account: https://discord.gg/s5DmUtWC
also I would contact discord support

**Comment by ghost** [comment]
ok, and tell them I've experienced this page while clearing my pending friends list.
![Screenshot (7)](https://user-images.githubusercontent.com/73761886/139111500-69ea6781-f852-4850-8a6a-9b4614e07a1c.png)

**Comment by ghost** [comment]
@quick007, I've contacted the discord that I've been falsely flagged, so they may remove the Verification for phone screen on my discord account and my IP Address.

**Comment by ghost** [comment]
I will lock this conversation, because the problem with discord has been solved.
